mkt/commonplace/templates/commonplace/index_react.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
```

Comment:
i'd just think of it as the app for serving frontend projects. there's a bit of stuff that carries over and we still need